src/test/overlay/reduce_relay_test.cpp

```diff
@@ -0,0 +1,1301 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright 2020 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+#include <ripple/basics/random.h>
+#include <ripple/beast/unit_test.h>
+#include <ripple/overlay/Message.h>
+#include <ripple/overlay/Peer.h>
+#include <ripple/overlay/Slot.h>
+#include <ripple/protocol/SecretKey.h>
+#include <boost/optional.hpp>
+#include <boost/thread.hpp>
+#include <numeric>
+#include <optional>
+#include <ripple.pb.h>
```

Comment:
Standard includes should come last. We need to move <ripple.pb.h> up. We also usually leave a space between the different include groups (usually groups: ripple boost std).